docs/UserGuide.md

```diff
@@ -217,40 +220,74 @@ Format: `user_guide`
 
 Format: `help` or `help [COMMAND]`
 
+* The `COMMAND` refers to any valid command that is implemented.
+
 Examples:
-* `help` Shows the list of commands.
+* `help` Shows the list of valid commands.
 * `help find` Shows the description of `find` command.
 * `help add` Shows the description of `add` command.
 
-Examples:
-* `find Mac` returns `MacDonald`
-* `find fairprice` returns `NTUC Fairprice` <br>
-
 ### Viewing a client : `view`
 
-`view` displays the remarks and transaction log of the specified client.
+Displays the remarks and transactions of the specified client.
 
 Format: `view INDEX`
 
 * Views the client at the specified `INDEX`.
-* The index refers to the index number shown in the displayed client list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* The `INDEX` refers to the index number shown in the displayed client list.
+* The `INDEX` **must be a positive integer** 1, 2, 3, …​
+
+### Getting the User Guide: `user_guide`
+
+Returns the url to this user guide.
+
+Format: `user_guide`
+
+* Displays a pop-up that contains the url to this user guide.
+
+Examples:
+* `user_guide` return the url of this user guide.
 
 ### Sorting the address book: `sort`
 
-Coming soon.
+Sorts the specified client's transaction by either the latest transaction or oldest transaction.
+
+Format: `sort INDEX latest` or `sort INDEX oldest`
```

Comment:
Is it possible to follow the format given above, where `CAPS` denote something that the user has to input, 
`Format: sort INDEX ORDER`?
Then below specify that ORDER must be either `latest` or `oldest`.

The same is applicable for the `filter` part of the UG

Reply:
thanks! will update in the UG